fsevents.js

```diff
@@ -6,6 +6,13 @@
 /* jshint node:true */
 'use strict';
 
+if (process.platform !== 'darwin') {
+  var err = new Error('Cannot find module \'fsevents\'');
```

Comment:
I saw your comment but I still think the error message is misleading.  A user looking at the stack trace will see it originating from the fsevents module and - if their mind works anything like mine - be mighty confused.